src/main/java/software/amazon/cloudformation/LambdaWrapper.java

```diff
@@ -126,7 +127,7 @@ protected LambdaWrapper() {
         this.typeReference = getTypeReference();
     }
 
-    /**
+    /*
```

Comment:
Isn't JavaDoc comment meant to start with `/**`? (There's a few of these changes in this PR)

Reply:
For some reason the comment kept failing on travis but successful locally. Will try to give it another shot.